src/common.c

```diff
@@ -46,6 +46,13 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 
+// fork / exec
```

Comment:
I think we nowhere else wrote which functions we import. So I would remove this here as well.